x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/perform_bulk_action_route.ts

```diff
@@ -286,15 +300,56 @@ export const performBulkActionRoute = (
           savedObjectsClient,
         });
 
+        const query = body.query !== '' ? body.query : undefined;
+
+        if (body.action === BulkAction.edit) {
```

Comment:
Any reason why this isn't just one of the `switch` conditions down below? Looks like it was moved up here, but there doesn't seem to be any reason it couldn't have stayed with the other action types.

Reply:
It might be worth adding a comment here to indicate why the `edit` action is separate from the others.

Reply:
added a comment. thanks for the suggestion